### ISSUE TITLE
关于cc.convertColor的BUG

### ISSUE THREAD

**Issue by shysean** [cocos2d/cocos2d-x#14071]
Cocos2d.lua 的方法 cc.convertColor

```
        if input.a then
            if math.ceil(input.a) ~= input.a or input.a >= 1 then
                ret.a = input.a * 255
            else
                ret.a = input.a
            end
        else
            ret.a = 255
        end
```

是不是应该把条件 input.a >= 1 改为 input.a <= 1 ???

**Comment by zilongshanren** [comment]
@shysean  
Yeah, I think you are right.

@dualface  @samuele3hu  Could you please take a look at this issue?

**Comment by dualface** [comment]
@shysean You are right.

**Comment by minggo** [comment]
Where is `Cocos2d.lua`, i can not find it.

**Comment by minggo** [comment]
Oh, found it.

**Comment by minggo** [comment]
fixed in #16047
